app/components/collectionShow/collectionPaperList.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { PaperInCollection } from '../../model/paperInCollection';
+import { CurrentUser } from '../../model/currentUser';
+import { CollectionShowState } from '../../containers/collectionShow/reducer';
+import { Collection } from '../../model/collection';
+import CollectionPaperItem from './collectionPaperItem';
+import ArticleSpinner from '../common/spinner/articleSpinner';
+import Icon from '../../icons';
+import { withStyles } from '../../helpers/withStylesHelper';
+import { removePaperFromCollection } from '../dialog/actions';
+import formatNumber from '../../helpers/formatNumber';
+import CollectionPapersControlBtns from './collectionPapersControlBtns';
+import { ACTION_TYPES } from '../../actions/actionTypes';
+const styles = require('./collectionPaperList.scss');
+
+interface CollectionPaperListProps {
+  itsMine: boolean;
+  papersInCollection: PaperInCollection[];
+  currentUser: CurrentUser;
+  collectionShow: CollectionShowState;
+  userCollection: Collection;
+  dispatch: Dispatch<any>;
+}
+
+const CollectionPaperInfo: React.FC<{ collectionShow: CollectionShowState }> = ({ collectionShow }) => {
+  return (
+    <div className={styles.subHeader}>
+      <div>
+        <span className={styles.resultPaperCount}>{`${formatNumber(collectionShow.papersTotalCount)} Papers `}</span>
+        <span className={styles.resultPaperPageCount}>
+          {`(${collectionShow.currentPaperListPage} page of ${formatNumber(collectionShow.totalPaperListPage)} pages)`}
+        </span>
+      </div>
+    </div>
+  );
+};
+
+const CollectionPaperList: React.FC<CollectionPaperListProps> = props => {
+  const { itsMine, papersInCollection, currentUser, collectionShow, userCollection, dispatch } = props;
+
+  const handleRemovePaperFromCollection = React.useCallback(
```

Comment:
this function should check the current user exists or not. (ex: use `checkAuthStatus`)
because the 'new tab' problem could happen here too.